server/service/src/pricing/item_price.rs

```diff
@@ -0,0 +1,84 @@
+use repository::RepositoryError;
+use repository::{
+    EqualFilter, MasterListFilter, MasterListLineFilter, MasterListLineRepository,
+    MasterListRepository,
+};
+
+use crate::service_provider::ServiceContext;
+
+pub struct ItemPriceLookup {
+    pub item_id: String,
+    pub customer_name_id: Option<String>, // Unused right now, but could be used to get discount for a specific name
+}
+
+#[derive(Debug, PartialEq)]
+pub struct ItemPrice {
+    pub item_id: String,
+    pub default_price_per_unit: Option<f64>,
```

Comment:
Went with everything optional inside the response, but always getting a pricing reponse from the backend.
Felt simpler in the frontend...

Reply:
Hmm, i think it's better as compulsory, I might be missing something but I feel like it's simpler for front end to have non optional

Reply:
I've added a comment here

Reply:
Thanks! Will try to remember for next time...